crates/lair_keystore/src/lib.rs

```diff
@@ -61,8 +62,16 @@ pub async fn execute_load_ed25519_keypair_from_yaml(
         store::spawn_entry_store_actor(config.clone(), store_file).await?;
 
     use std::fs::File;
-    let file = File::open(load_ed25519_keypair_from_yaml)?;
-    let keypair: entry::EntrySignEd25519 = serde_yaml::from_reader(&file)?;
+    let file = File::open(load_ed25519_keypair)?;
+    let blob = BufReader::new(file).lines()
+        .map(|line| line.and_then(|v| v.parse().map_err(|e| Error::new(ErrorKind::InvalidData, e))))
+        .collect::<Result<Vec<u8>, Error>>()?;
+
+    let keypair = entry::EntrySignEd25519 {
+        priv_key: lair_keystore_api::internal::sign_ed25519::SignEd25519PrivKey::from(blob[64..].to_vec()),
+        pub_key: lair_keystore_api::internal::sign_ed25519::SignEd25519PubKey::from(blob[32..].to_vec()), 
```

Comment:
I don't think we want the private key in there too : )

```suggestion
        pub_key: lair_keystore_api::internal::sign_ed25519::SignEd25519PubKey::from(blob[32..64].to_vec()), 
```